docs/develop.md

```diff
@@ -229,17 +229,9 @@ publications.
 
 These instructions are intended for core developers.
 
-- Create a new branch, e.g. `prep_release_vXXX`
-- Update "version" in `src/wam2layers/__init__.py` (use [semantic versioning](https://semver.org/))
-- Make sure the code is neatly formatted (see above)
-- Make sure the changelog is up to date; update the "Unreleased" header with the
-  new version, and add a new "Unreleased" header above
-- Review and merge the release branch
-- Check on Zenodo whether the Zenodo integration with WAM2layers/WAM2layers is switched to ON. In case the switch is OFF, do not turn it ON, but contact @ruudvdent to turn it ON for you. This is in order to avoid creating multiple Webhooks and thus multiple releases.
-- [Make a release on GitHub](https://docs.github.com/en/repositories/releasing-projects-on-github/managing-releases-in-a-repository)
-- Upon release, the [Github Actions workflow for publishing](https://github.com/WAM2layers/WAM2layers/actions/workflows/publish.yaml) will run. More extensive documentation on publishing is available [here](https://packaging.python.org/en/latest/guides/publishing-package-distribution-releases-using-github-actions-ci-cd-workflows/).
-- Check that the new release is successfully rendered on Zenodo and PyPI.
-- Verify that you can install the new version with pip.
+- Create a new issue using the issue template `new_release`
+- Follow the steps in this issue template
+- After the release close the associated issue
```

Comment:
I've added the "close the issue" step to the template. This doesn't really need to be a bullet list anymore, right? Also we might insert a direct link.

```suggestion
To make a new release, create an issue using the issue template [`New release`](https://github.com/WAM2layers/WAM2layers/issues/new?template=new-release.md) and follow the steps outlined in this issue.
```